src/test/bundle_test.ts

```diff
@@ -35,7 +37,7 @@ class FileTransform extends Transform {
     this.transform = transform;
   }
   _transform(f: File, _encoding: string, cb: () => void) {
-    this.transform(this, f);
+    this.transform(this, f.clone());
```

Comment:
This is a meaningful change in behavior though, especially with the potential forest of streams we'll encounter. Can a test fail without this?